docs/tutorials/building_public_actors.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Building public actors
+description: Find out the flow for building, publishing, and maintaining public actors on the Apify platform.
+menuWeight: 3.3
+paths:
+    - tutorials/building-public-actors
+---
+
+# Building public actors
+
+There are four main stages of building a public actor:
+
+1. [Development]({{@link actors/development.md}}).
+2. [Publication]({{@link actors/publishing.md}}).
+3. [Testing and maintenance]({{@link actors/development/testing_and_maintenance.md}}).
+4. [Promotion]({{@link actors/publishing/seo_and_promotion.md}}).
+
+While you don't have to maintain private actors, public actors are a more long-term project. Be prepared to stick with the project for at least a few months. Ensure you have enough time to maintain the project (~2 hours weekly). With this perspective, you will be able to monetize and promote your actors better.
+
+Code quality is a priority. It is public on [Github](https://github.com), and your actors may be a new user's first and critical contact with Apify. Code should be open to changes and updates – don't fear to refactor. Beware of breaking changes, though! If you do a breaking change, communicate it to all existing users with an explanation.
```

Comment:
This is not true, you can have a public actor with private code (e.g. paid actors).